e2e/protractor.conf.js

```diff
@@ -1,14 +1,16 @@
+// @ts-check
 // Protractor configuration file, see link for more information
 // https://github.com/angular/protractor/blob/master/lib/config.ts
 
-const {
-  SpecReporter
-} = require('jasmine-spec-reporter');
+const { SpecReporter } = require('jasmine-spec-reporter');
 
+/**
+ * @type { import("protractor").Config }
+ */
 exports.config = {
   allScriptsTimeout: 11000,
   specs: [
-    './e2e/**/*.e2e-spec.ts'
+    '../e2e/**/*.e2e-spec.ts'
```

Comment:
Maybe we could just do:
```suggestion
    './**/*.e2e-spec.ts'
```